python/src/SobolSimulationAlgorithm_doc.i.in

```diff
@@ -24,20 +24,34 @@ The algorithm can operate on a multivariate model :math:`f`, in this case it ope
 
 Several estimators are available (Saltelli, Jansen, ...).
 
-The algorithms stops when, on all components, first and total order indices haved been estimated with enough precision or the first order indices are
-*separable* from the total order indices:
-
+Let us denote by $n_X$ the number of input variables. 
+For any $j=1,...,n_X$, let us denote by $\Phi_j^F$ (resp. $\Phi_j^T$) the cumulated distribution 
+function of the gaussian asymptotic distribution of the estimator 
+of the Sobol' first (resp. total) order indice.
+Let $\alpha\in[0,1]$ be the level of the confidence interval and $\epsilon\in(0,1]$ the 
+length of this confidence interval. 
+The algorithms stops when, on all components, at least one of the following conditions 
+is satisfied: 
+- first and total order indices haved been estimated with enough precision or  
+- the first order indices can be *separated* from the total order indices.
+
+The precision is said to be sufficient if the length of the $\alpha$-level confidence interval 
+is smaller than $\epsilon$:
 .. math::
+    (\Phi_j^F)^{-1}(1-\alpha/2) - (\Phi_j^F)^{-1}(\alpha/2) \leq \epsilon \textrm{ and } 
+    (\Phi_j^T)^{-1}(1-\alpha/2) - (\Phi_j^T)^{-1}(\alpha/2) \leq \epsilon
 
-    \forall{j}
-
-    \Phi^{F}_j(1-\alpha)-\Phi^{F}_j(\alpha) \leq \epsilon \cap \Phi^{T}_j(1-\alpha)-\Phi^{T}_j(\alpha) \leq \epsilon
+for all $j=1,...,n_X$. 
 
-    or
+The first order indices can be *separated* from the total order indices if:
+.. math::
 
     \Phi^{T}_j(\alpha) \ge \Phi^{F}_j(1-\alpha)
 
+for all $j=1,...,n_X$. 
```

Comment:
you need a need line before math::, and the "for all" must be indented

Reply:
Ok : this is fixed.